integration-tests/opentelemetry-jaeger-remote/src/test/java/io/quarkus/it/opentelemetry/jaeger/OTelJaegerRemoteIT.java

```diff
@@ -0,0 +1,7 @@
+package io.quarkus.it.opentelemetry.jaeger;
+
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+
+@QuarkusIntegrationTest
```

Comment:
I assume this is meant to test things in native mode?

If so, then we'll need to configure the proper native JSON config file in the .github directory

Reply:
This is quite specific. Probably no need. There are other IT related projects for OTel that are not running on Native.

Reply:
👌